library/std/src/sys/personality/dwarf/eh.rs

```diff
@@ -43,6 +44,7 @@ pub struct EHContext<'a> {
     pub get_data_start: &'a dyn Fn() -> usize, // Get address of the data section
 }
 
+#[allow(non_camel_case_types)]
 pub enum EHAction {
```

Comment:
```suggestion
pub enum EhAction {
```
`Eh` would be fine in this context.